automated install/basic-install.sh

```diff
@@ -260,6 +260,41 @@ setStaticIPv4(){
 	fi
 }
 
+setDNS(){
+	DNSChoseCmd=(whiptail --separate-output --radiolist "Select DNS Servers" $r $c 2)
+	DNSChooseOptions=(Google "Use Google's DNS Servers" on
+					  DynDNS "Use DynDNS's DNS Servers" off)
+	DNSchoices=$("${DNSChoseCmd[@]}" "${DNSChooseOptions[@]}" 2>&1 >/dev/tty)
+
+	case $DNSchoices in
+		Google)
+			echo "Google selected."
+			piholeDNS1="8.8.8.8"
+			piholeDNS2="8.8.4.4"
+			;;
+		DynDNS)
+			echo "DynDNS selected."
+			piholeDNS1="208.67.222.222"
+			piholeDNS2="208.67.220.220"
+			;;
+	esac
+}
+
++versionCheckDNSmasq(){
+	# Check if /etc/dnsmasq.conf is from pihole.  If so replace with an original and install new in .d directory
+	dnsFile="/etc/dnsmasq.conf"
+	dnsSearch="addn-hosts=/etc/pihole/gravity.list"
+	
+	if grep -q $dnsSearch $dnsFile; then
+		$SUDO mv /etc/dnsmasq.conf /etc/dnsmasq.conf.orig
```

Comment:
Here, if dnsmasq.conf is a Pi-Hole conf then there already will be a dnsmasq.conf.orig, so we don't want to nuke the file. How about a 3 way switch? Copy dnsmasq.conf to dnsmasq.conf.pihole, then copy the dnsmasq.conf.orig back to dnsmasq.conf...